### DISCUSSION TITLE
How to change ImGui::Text to multiline within a table and row if column is too narrow

### DISCUSSION THREAD

**Discussion by peterwilson136** [ocornut/imgui#3846] [Old/Archived discussions]
I have a table in ImGui, and I have some text under a column title "Description", the way the text is displayed is 
`ImGui::Text("This is some text");` but If a user changes the text to be bigger (edit and add more) I want the column to not keep growing in width, but rather only grow in width if they resize the column. I want the text to wrap within the confines of the width of the column it lies in and move to a new line accordingly. How can I do this?

**Comment by rokups** [comment]
What table/column flags did you use? Did you try `ImGuiTableFlags_SizingFixedFit`?

**Reply by ocornut** [reply]
And you can use `ImGui::TextWrapped()` to wrap text to contents width.
All of those are shown in the tables demo.
